src/Attribute/Mapper.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AutoMapper\Attribute;
+
+use AutoMapper\ConstructorStrategy;
+
+/**
+ * Configures a mapper.
+ */
+#[\Attribute(\Attribute::IS_REPEATABLE | \Attribute::TARGET_CLASS)]
+final readonly class Mapper
+{
+    public function __construct(
+        public ?string $source = null,
+        public ?string $target = null,
```

Comment:
Shouldn't we change this property names to `acceptedSource` / `acceptedTarget` ? To not be confused with `MapFrom` & `MapTo` properties that are not in the same context.

Reply:
Also I'm not sure we should accept both properties being set, isn't it a bit confusing ?